modules/kubernetes-addons/adot-collector-java/locals.tf

```diff
@@ -0,0 +1,60 @@
+locals {
+  name                           = "adot-collector-java"
+  adot_collector_service_account = "adot-collector-java"
+
+  default_helm_config = {
+    name        = local.name
+    repository  = null
+    chart       = "${path.module}/otel-config"
```

Comment:
I didn't do a full comparison but just curious why the local/custom helm chart and not the one provided by opentelemetry? https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts